doc/paradox/sql-to-slick.md

```diff
@@ -155,7 +155,19 @@ Scala's equivalent for `SELECT` is `map`. Columns can be referenced similarly an
 @@snip [SqlToSlick.scala](../code/SqlToSlick.scala) { #sqlQueryFilter }
 
 <h4>Slick</h4>
-Scala's equivalent for `WHERE` is `filter`. Make sure to use `===` instead of `==` for comparison.
+Scala's equivalent for `WHERE` is `filter`. There are also the convenience variations `filterIf` and `filterOpt`.
```

Comment:
```suggestion
```suggestion
Scala's equivalent for `WHERE` is
@scaladoc[`filter`](scaladoc:slick.lifted.Queryl#filter[T](f:E=>T)(implicitwt:slick.lifted.CanBeQueryCondition[T]):slick.lifted.Query[E,U,C]).
There are also the convenience variations
@scaladoc[`filterIf`](slick.lifted.Query#filterIf[T](p:Boolean)(f:E=%3ET)(implicitevidence$2:slick.lifted.CanBeQueryCondition[T]):slick.lifted.Query[E,U,C])
and
@scaladoc[`filterOpt`](slick.lifted.Query#filterOpt[V,T](optValue:Option[V])(f:(E,V)=>T)(implicitevidence$1:slick.lifted.CanBeQueryCondition[T]):slick.lifted.Query[E,U,C]).
```

Reply:
I hope the links work

Reply:
Incorporated. Is it possible to check the built version?

Reply:
If you want to check locally just run `sbt paradox`. Otherwise we can just merge it.

Reply:
Or `sbt "show paradox"`, then it will print where the generated docs are

Reply:
Or just run scaladocCheckLinks. I should add that to CI.

Reply:
I just tried this out, got couple of errors, I think due to the encoding of `=>`, fixed them as `=>%3E` and it worked. Though I think scaladoc needs a little work to make the selected method a little more prominent :D I can see that its description is expanded.